src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.tsx

```diff
@@ -26,7 +26,7 @@ import { Datepicker } from '@shared/components/Datepicker';
 import { Timepicker } from '@shared/components/Timepicker';
 import { OPTIONAL_LABEL } from '@shared/const';
 import { toastService } from '@shared/services';
-import { visitsService } from '@visits/services';
+import { VisitsService } from '@visits/services';
```

Comment:
Best consistent lower- of upper-camelcase gebruiken

Reply:
de VisitsService heeft static methoden waardoor je ze aanroept op de class:
```
VisitService.create();
```
de toastService heeft gewone methoden waardoor je ze moet aanroepen op een instance van de class
```
const toastService = new ToastService();
toastService.notify();
```

We moeten bepalen welke paradigm we liefst gebruiken en dan alle services op dezelfde manier doen.
Ik gebruik graag de static methods, omdat je dan nooit de kans hebt om 2 instances van dezelfde klasse te maken. En dat het ook niet de bedoeling is om state in de service te bewaren. Welke approach verkiezen jullie? @ian-emsens-shd @BavoVdg ?